meta-integrity/lib/oeqa/runtime/ima.py

```diff
@@ -72,7 +72,10 @@ def test_ima_signature(self):
 
     def test_ima_overwrite(self):
         ''' Test if IMA prevents overwriting signed files '''
-        signed_file = "/bin/sh"
+        signed_file = "/bin/arping"
```

Comment:
You also seem to get test failure if the file is not found. Be careful hard coding the binary. Perhaps pick something from /bin that really exists.